desktop/cmp/input/CodeInput.scss

```diff
@@ -93,6 +100,7 @@
   }
 
   &__toolbar {
+    background-color: var(--xh-bg-alt);
```

Comment:
This is deliberately matching the gutter color (as opposed to standard toolbar color), which I think looks a bit more integrated / subtle, especially with the toolbar in compact mode.